lib/network/api/tx_operations.go

```diff
@@ -0,0 +1,62 @@
+package api
+
+import (
+	"fmt"
+	"net/http"
+
+	"boscoin.io/sebak/lib/block"
+	"boscoin.io/sebak/lib/common/observer"
+	"boscoin.io/sebak/lib/network/api/resource"
+	"boscoin.io/sebak/lib/network/httputils"
+	"boscoin.io/sebak/lib/storage"
+	"github.com/gorilla/mux"
+)
+
+func (api NetworkHandlerAPI) GetOperationsByTxHashHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
how about move content of this file to operation.go?

Reply:
@kfangw  But This makes a long line file.. Is it?  We need to make another handler .. (`/transactions/{hash}/operation/{hash}`) I thought that `tx_operations.go` and `tx_operation.go` . 
If you want to `operation.go` has all operation related handlers. I am OK :->